src/Utils/utils.ts

```
@@ -544,3 +544,13 @@ export const fahrenheitToCelsius = (fahrenheit: number) => {
 export const keysOf = <T extends object>(obj: T) => {
   return Object.keys(obj) as (keyof T)[];
 };
+
+export const handleNegativeValue = (
+  event: React.FormEvent<HTMLInputElement>,
+): void => {
+  const input = event.currentTarget;
+  input.value = input.value.replace(/-/g, "");
+  if (isNaN(Number(input.value))) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)